CMakeLists.txt

```diff
@@ -689,7 +689,7 @@ if(UNIX)
     endif()
 
     execute_process(COMMAND ${CMAKE_COMMAND} -E remove_directory "${PROJECT_BINARY_DIR}/${ENERGYPLUS_PATH}")
-
+    file(REMOVE "${PROJECT_BINARY_DIR}/python/engine/pip_install_done.stamp")
```

Comment:
When we change E+ directory (update), remove the .stamp file to trigger another install